client/verta/verta/monitoring/profiler.py

```diff
@@ -204,9 +204,9 @@ def profile_column(self, df, column):
             bins = self._bins[column]
         else:
             bins = self._bins
-        values, limits = self._np.histogram(df[column], bins=bins)
-        values = [v.item() for v in values]
-        limits = [lim.item() for lim in limits]
+        values, limits = self._np.histogram(df[column].dropna(), bins=bins)
```

Comment:
`dropna()` omits missing values, which prevents `np.histogram()` from failing due to NaNs, and also matches the behavior of `BinaryHistogramProfiler` (which also omits missing values when profiling).